WeakAuras/WeakAuras.lua

```diff
@@ -1170,6 +1182,10 @@ function Private.Login(initialTime, takeNewSnapshots)
       coroutine.yield();
     end
 
+
+    Private.Features:Hydrate()
```

Comment:
Hydration of feature flags is timed so that it should be the first thing that happens on PLAYER_LOGIN (the bit of code above is a saved variables migration that would only happen if a user played for a brief period in 2019 but then quit until now). So, even if a user opts into a persisted feature, the code will still briefly think the feature is disabled.